inc/commonitiltask.class.php

```diff
@@ -1543,7 +1541,7 @@ class='calendrier'>";
                                     ['title'  => __('Availability')]);
 
       echo "<br />";
-      echo Html::image($CFG_GLPI['root_doc']."/pics/group.png")."&nbsp;";
+      echo "<i class='fas fa-users'></i>&nbsp;";
```

Comment:
Since this one is decorative only, add a ` aria-hidden='true'`

Reply:
Thanks, I just added `aria-hidden` as the "true" is not mandatory